README.md

```diff
@@ -1 +1,250 @@
-# cli
\ No newline at end of file
+<div align="center">
+  <img alt="go-octopusdeploy Logo" src="https://user-images.githubusercontent.com/71493/133961475-fd4d769f-dc32-4723-a9bd-5529c5b12faf.png" height="140" />
+  <h3 align="center">cli</h3>
+  <p align="center">Command Line Interface for <a href="https://octopus.com/">Octopus Deploy</a> 🐙</p>
+  <p align="center">
+    <a href="https://github.com/OctopusDeploy/cli/releases/latest"><img alt="GitHub release" src="https://img.shields.io/github/v/release/OctopusDeploy/cli.svg?logo=github&style=flat-square"></a>
+    <a href="https://goreportcard.com/badge/github.com/OctopusDeploy/cli"><img src="https://goreportcard.com/badge/github.com/OctopusDeploy/cli" alt="Go Report"></a>
+  </p>
+</div>
+
+---
+
+## Overview
+
+This project aims to create a new CLI for communicating with the Octopus Deploy Server, written in Go.
+
+It does **not** seek to be a drop-in replacement for the existing CLI which is written in C# using .NET.
+https://github.com/OctopusDeploy/OctopusCLI
+
+### Differences from the .NET CLI
+
+The new CLI will not initially contain all the features of the existing .NET CLI. 
+Over time we plan to add features and may eventually reach parity, but our intent is that both the 
+.NET and Go CLI's will co-exist for a significant period of time.
+
+The New CLI restructures the command line to be more consistent, and fit with convention
+across other popular CLI apps. It is built on the popular and widely-used [Cobra](https://github.com/spf13/cobra) 
+command line processing library.
+
+#### Examples:
+
+**.NET CLI**
+
+    octo list-releases 
+    octo create-release
+
+**Go CLI**
+
+    octopus release list 
+    octopus release create
+
+The new CLI supports an "interactive" mode, where it will prompt for input where
+parameters are not fully specified on the command line.
+
+## Documentation
+
+End-user documentation will be provided via the octopus documentation site at a future date.
+
+## 🤝 Contributions
+
+Contributions are welcome! :heart: Please read our [Contributing Guide](CONTRIBUTING.md) for information about how to get involved in this project.
+
+# Developer Guide
+
+## Getting Started
+
+First, ensure that you have [Go](https://go.dev/) installed, and available in your `PATH`.
+To verify this, open a new terminal window and type `go version`. You should see something similar to `go version go1.18.4 windows/amd64`
+
+Next, clone this git repository
+
+Next, open the directory you cloned into, navigate into the `cmd/octopus` directory, and type `go build .`
+
+```shell
+cd <your-local-development-dir>
+git clone https://github.com/OctopusDeploy/cli
+cd cli
+cd cmd/octopus
+go build .
+```
+
+If successful, the go compiler does not output anything. You should now have an `octopus` binary 
+(`octopus.exe` on windows) in your current directory.
+
```

Comment:
We could also mention that they could run `$ make` from the command line/terminal and the default action is to generate the binary via `go build`.